### DISCUSSION TITLE
I cannot receive student benefits

### DISCUSSION THREAD

**Discussion by byalexandrepedrosa** [community/community#68346] [GitHub Education]
### Hi everyone,

Months after got it removed, and problem still same, seems GitHub is going from bad to worse.

Support dont allow more send mails, and dont answer, recently did a data leak:

[https://cybernews.com/security/microsoft-ai-research-team-github-leak/](https://cybernews.com/security/microsoft-ai-research-team-github-leak/)

Someday someone from support will check messages here and fix my issue?

On my situatuin validation always fails with message:

Your application cannot be reviewed until you fix the following:
Your image should be one of the following:
Dated school ID

The data is on documents, but not in USA format, its on Brazilian format DD/MM/YYYY if you guys automate the date read and cont consider other formats, this is an big mistake.

**Comment by byalexandrepedrosa** [comment]
Tell me an email to send required data...

Publishing personal data publicly is harmful, my proof of registration shows the equivalent of the US Social Number, in Brazil it is CPF.

**Comment by samus-aran** [comment]
Hi @byalexandrepedrosa 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

We are seeing a higher volume of student developer pack inquiries coming through. We are working our way through addressing these as fast as possible. I can share that your request has been passed onto the GitHub Education team and will be processed as soon as possible, but I am afraid I do not have a timeline to share for when you can expect this to happen. If I have an update we'll let you know here or in your ticket 👍 

Regards to your submission the date format should not hinder it as we do accept multiple formats, so I suspect it is something else. We do have a preference of evidence with Student ID and Academic Email being the most preferred but also accept other forms of evidence depending on the school applying for and other students.

**Reply by byalexandrepedrosa** [reply]
Thanks for the reply!!!

This is the first time someone give an feedback about my issue.

Unfortunately, the institution where I study does not offer any support to students, there are no emails, there is no Student ID.

University Amply here in Brazil focuses on offering one of the cheapest degrees with monthly fees, and to achieve this they do so with degrees in distance learning (online) and without any support.

I will also send a photo of the portal on the ticket where you can check my personal details.
